src/pex/pex.c

```diff
@@ -92,9 +116,13 @@ uint8_t read_register(pex_t* pex, uint8_t addr) {
     return ret;
 }
 
-// Sets the direction of pin 'pin' on bank 's' to state 'dir'
-// bank - A (GPIOA) or B (GPIOB)
-// direction - OUTPUT or INPUT
+/*
+Sets the direction of pin 'pin' on bank 's' to state 'dir'
+pex - pointer to the pex device
+pin - the pin that is having its direction set
+s - the bank the pin is on, A (GPIOA) or B (GPIOB)
+dir - desired direction: OUTPUT or INPUT
+*/
 void pex_set_pin_dir(pex_t* pex, uint8_t pin, pex_bank_t s, pex_dir_t dir) {
```

Comment:
Should switch the order of pin and s (I know from experience it's confusing when using it), and give s a better name